spec/s3_meta_sync_spec.rb

```diff
@@ -345,7 +353,7 @@ def call(*args)
   end
 
   describe "CLI" do
-    let(:params) { "--key #{config[:key]} --secret #{config[:secret]} --region #{config[:region]}" }
+    let(:params) { "--key #{config[:key]} --secret #{config[:secret]} " }
```

Comment:
With regions its really bad thing, such URLs like this in code work only for us standard region.
In file creditianals.yml I had to comment out #:region: us-east-1 and put my bucket in standard region. http://docs.aws.amazon.com/general/latest/gr/rande.html

 19   def download(file)
 20     url=URI.parse("https://s3.amazonaws.com/#{config[:bucket]}/#{file}")
 21     open(url).read
 22   rescue